### PR TITLE
Document the CST API according to OpenAPI spec

### PR DESCRIPTION
**Pull request by nettoclaudio**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,23 +15,33 @@ CST_CERTS_DIR ?= .certs
...
 	grep -v "mock.go" $(COVERAGE_FILE) > coverage.txt
 	$(GO) tool cover -func=coverage.txt
 
+validate-swagger-spec:
+	$(SWAGGER) validate swagger.yml
+
 get-dev-deps:
-	$(GO) get -u golang.org/x/lint/golint
 	$(GO) get -u github.com/golang/dep/cmd/dep
+	$(GO) get -u golang.org/x/lint/golint
```
Comment (ggarnier) [inline Makefile:43]: I suggest that you use https://github.com/golangci/golangci-lint

**Review by ggarnier — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (andrestc) [reply Makefile]: :+1:

File: swagger.yml [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+          properties:
+            image:
+              type: "string"
+              example: "tsuru/cst:latest"
+
+      responses:
+        201:
+          description: "Scan successfully scheduled"
+        204:
+          description: "Scan ignored because already exist a scheduled scan"
```
Comment (andrestc) [inline swagger.yml]: I would write this as:  Scan ignored because a scheduled scan already exists

File: swagger.yml [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+        500:
+          description: "Problem to register the scan on database service."
```
Comment (andrestc) [inline swagger.yml]: I think "Failed to register the scan on database service." reads better

**Review by andrestc — APPROVED** [review]
Looks good, made a few suggestions but can merge as is

File: Makefile (hunk 1 above)
Comment (nettoclaudio) [reply Makefile:43]: It's a really good suggestion. I'll replace golint in another PR.

**Pull request merged**